packages/pmd.rb

```diff
@@ -3,45 +3,45 @@
 class Pmd < Package
   description 'An extensible cross-language static code analyzer.'
   homepage 'https://pmd.github.io/'
-  version '6.28.0'
+  version '6.52.0'
   license 'custom'
   compatibility 'all'
-  source_url 'https://github.com/pmd/pmd/releases/download/pmd_releases%2F6.28.0/pmd-bin-6.28.0.zip'
-  source_sha256 '9a19365f2e107ae801b39be04c5c03cdca2d352c450faac639a6dd95b5c3ab0c'
+  source_url 'https://github.com/pmd/pmd/releases/download/pmd_releases%2F6.52.0/pmd-bin-6.52.0.zip'
+  source_sha256 'fa757430ee45bb79ce3d10bb5706b71aa7bd2f0df40c135b1ae2ffd2ae91810e'
 
-  binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pmd/6.28.0_armv7l/pmd-6.28.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pmd/6.28.0_armv7l/pmd-6.28.0-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pmd/6.28.0_i686/pmd-6.28.0-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pmd/6.28.0_x86_64/pmd-6.28.0-chromeos-x86_64.tar.xz'
-  })
-  binary_sha256({
-    aarch64: '3ffa1b5c562929b74903a42bb8246203b6b1a09a156031e25522991bb25eea51',
-     armv7l: '3ffa1b5c562929b74903a42bb8246203b6b1a09a156031e25522991bb25eea51',
-       i686: '8002bc791ac2751dca2c557b248a3d52eff48c5ad04d8ad4320fc4b19a88a230',
-     x86_64: 'a9595a5dc740d9f952526817488a86bca488ec06daac8b990b724eca45459629'
-  })
-
-  depends_on 'jdk8'
+  depends_on 'openjdk8'
   depends_on 'unzip'
 
+  no_compile_needed
+
+  def self.build
+    cpd = <<~EOF
+      #!/bin/bash
+      PWD=$(pwd)
+      cd #{CREW_PREFIX}/share/pmd
+      bin/run.sh cpd "$@"
+      cd $PWD
```

Comment:
We don't need to `cd` back since this is independent from the shell process

Reply:
Good call.  Fixed.